stable/appmesh-controller/README.md

```diff
@@ -86,11 +86,48 @@ helm upgrade -i appmesh-controller eks/appmesh-controller \
 
 The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
-**Note:** When using IRSA, make sure the Envoy proxies have the following IAM policies attached for Envoy to authenticate with AWS App Mesh and fetch it's configuration
+**Note**
+Make sure that the Envoy proxies have the following IAM policies attached for the Envoy to authenticate with AWS App Mesh and fetch it's configuration
 - https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
 
-#### Setup IAM permissions manually on worker nodes
-If not setting up IAM role for service account, apply the IAM policies to your worker nodes:
+There are **2 ways** you can attach the above policy to the Envoy Pod  
+#### With IRSA     
+Download the Envoy IAM polocy  
+```
+curl -o envoy-iam-policy.json https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
+```
+
+Create an IAM policy called AWSAppMeshEnvoyIAMPolicy
+```
+aws iam create-policy \
+    --policy-name AWSAppMeshEnvoyIAMPolicy \
+    --policy-document file://envoy-iam-policy.json
+```
+
+Take note of the policy ARN that is returned
+
+Create Application namespace to which Envoy proxy will get injected
```

Comment:
I would say we don't need this step. Instead we can directly jump to the next step and ask them to fill in the relevant application namespace(s). We know that a Mesh can be spread across multiple namespaces and so this is required across all the namespaces that the mesh selects (or) their application is running.

Reply:
Got it. Will update this